docs/src/forc/commands.md

```diff
@@ -0,0 +1,16 @@
+Here are a list of commands available to forc:
+
+- [forc init](./forc_init.md)
+- [forc build](./forc_build.md)
+- [forc update](./forc_update.md)
+- [forc test](./forc_test.md)
+- [forc run](./forc_run.md)
+- [forc deploy](./forc_deploy.md)
+- [forc fmt](./forc_fmt.md)
+- [forc parse-bytecode](./forc_parse-bytecode.md)
+- [forc addr2line](./forc_addr2line.md)
+- [forc clean](./forc_clean.md)
+- [forc completions](./forc_completions.md)
+- [forc explorer](./forc_explorer.md)
+- [forc json-abi](./forc_json-abi.md)
+- [forc lsp](./forc_lsp.md)
```

Comment:
If the plan is to have these in their own sub-chapter (section??), then please put them into a `forc/commands/` directory, and rename `forc/commands.md` to `forc/commands/index.md`